internal/exec/docs_generate.go

```
@@ -0,0 +1,387 @@
+			subDir := filepath.Dir(path)
+
+			// Check if there's a local atmos.yaml in subDir
+			localAtmos := filepath.Join(subDir, "atmos.yaml")
+
+			// If found, we do an os.Chdir + re-init config from that location
+			if fileExists(localAtmos) {
+				oldDir, err := os.Getwd()
+				if err != nil {
+					return err
+				}
+				defer os.Chdir(oldDir) // ensure we go back no matter what
```

golangci-lint (1.62.2)

Error return value of `os.Chdir` is not checked

(errcheck)